### DISCUSSION TITLE
Importing *.nb, *.wls, or *.wl

### DISCUSSION THREAD

**Discussion by amindanial** [WLJSTeam/wljs-notebook#77] [General]
Can *.nb, *.wls, or *.wl be imported directly into WLJS?

**Comment by userrand** [comment]
I think this is similar to this issue https://github.com/JerryI/wolfram-js-frontend/issues/70#issue-2068835931 .

**Comment by jp-tollenboom** [comment]
If you have a Wolfram Language Package file (a .wl or a .m file), you can do this:
Get[mypakage.m].
This will load the package into the notebook session.

**Comment by JerryI** [comment]
We added experimental support for .md and .nb files.
